doc/aws.md

```diff
@@ -77,7 +77,9 @@ dpdk-devbind.py -s
 
 ## 5. Run the application
 
-Refer to [run.md](./run.md) section 3.2.
+Refer to [run.md](./run.md) after section 3.2.
+
+The ENA HW does not support RSS hash fields modification, please run `RxTxAPP` with args `--rss_mdoe l3_l4`.
```

Comment:
Can we auto assign rss to l3 and l4 for ENA like below?
```
    /* enable rss if no flow support */
    if (inf->flow_type == MT_FLOW_NONE && inf->rss_mode == MT_RSS_MODE_NONE) {
      if (mt_if(impl, port)->drv_type == MT_DRV_ENA)
        inf->rss_mode = MT_RSS_MODE_L3_L4; /* only rss l3 and l4 support */
      else
        inf->rss_mode = MT_RSS_MODE_L3_L4_DP_ONLY;
   }
```

Reply:
sure, thanks